### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by scpaye506**
First part of the dashboard page

Still need to add the update profile page in a separate PR

This breaks all of the dummy article and session lists, so I've made it just return an empty array for now until we get real data

### CODE REVIEW

File: app/assets/stylesheets/redesign/components/toggle.sass [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  display: inline-block
+  width: 48px
+  height: 25px
+
+.Toggle input
+  opacity: 0
+  width: 0
+  height: 0
+
+.Toggle-slider
```
Comment (andylushman) [inline app/assets/stylesheets/redesign/components/toggle.sass]: Looks like the toggle slider is backwards from what is in the mock. For example, in the mock its teal when on the left vs on the right, as it is here.

File: app/views/dashboards/show.html.slim [hunk 2]
```diff
@@ -1,40 +1,122 @@
...
+          li.Dashboard-card-list-item
+            = render partial: 'components/article_card', locals: { article: article }
+    - if @articles.blank?
+      .Dashboard-empty-message You have not submitted any content for this year.
+
+  section.Dashboard-section
+    .Dashboard-header-wrapper
+      h1.Dashboard-header My Sessions
+      - if AnnualSchedule.cfp_open?
+        a.Dashboard-header-action href="#{new_submission_path}"
```
Comment (andylushman) [inline app/views/dashboards/show.html.slim]: This currently throws an error but I'm guessing it something that needs to be taken care of by Jay?

File: app/assets/stylesheets/redesign/components/toggle.sass (hunk 1 above)
Comment (scpaye506) [reply app/assets/stylesheets/redesign/components/toggle.sass]: yeah, good notice. It's pretty standard for the toggle to be off when it's left and on when it's right, which is opposite of the designs. Was going to ask Geoff about it at some point. Not too worried about it not matching the design atm though.

File: app/views/dashboards/show.html.slim (hunk 2 above)
Comment (scpaye506) [reply app/views/dashboards/show.html.slim]: what error is it throwing?

File: app/assets/stylesheets/redesign/components/toggle.sass (hunk 1 above)
Comment (andylushman) [reply app/assets/stylesheets/redesign/components/toggle.sass]: Agreed. My first thought was that the design was off.

File: app/views/dashboards/show.html.slim (hunk 2 above)
Comment (andylushman) [reply app/views/dashboards/show.html.slim]: never-mind. It was because I didn't have redis running. Working now 👍

**Pull request merged**